src/main/java/winter/task/Task.java

```diff
@@ -0,0 +1,66 @@
+package winter.task;
+public class Task {
+    boolean isMarked;
+    int order;
+    String taskName;
+
+    public String doneCheckbox ="[ ]";
```

Comment:
This seems to be repeated in line 31 as well, maybe consider using a constant variable to store `"[ ]"` and have doneCheckbox reference that constant for its initial value?